homeassistant/components/tesla/binary_sensor.py

```diff
@@ -55,3 +55,7 @@ def is_on(self):
         _LOGGER.debug("Updating sensor: %s", self._name)
         await super().async_update()
         self._state = self.tesla_device.get_value()
+        try:
+            self._attributes = self.tesla_device.attrs
+        except AttributeError:
```

Comment:
Why would there be an attribute error?

Reply:
The underlying tesla_device may not have a attrs attribute or property.

Reply:
Ok I removed the try.